salt/states/ssh_known_hosts.py

```diff
@@ -12,6 +12,7 @@
         - present
         - user: root
         - fingerprint: 16:27:ac:a5:76:28:2d:36:63:1b:56:4d:eb:df:a6:48
+        - fingerprint_hash_type: md5
```

Comment:
I'm fine having only a sha256 example. I'll give it a shot. I'll try to get my own code using the sha256 hast type and update this if I get that working. Give me a couple days, please.